test/imperative.js

```diff
@@ -192,3 +192,17 @@ metatests.testSync('testAsync must return a test', (test) => {
   test.ok(t);
   t.end();
 });
+
+metatests.test('Must catch unhandledExeptions', (test) => {
+  const t = new metatests.ImperativeTest('Throwing test', () => {
+    throw new Error();
```

Comment:
What do you think of also adding assertions to check if the error caught by the test is the same error that was thrown inside the test?